src/fastoad/model_base/openmdao/tests/test_group.py

```diff
@@ -0,0 +1,179 @@
+#  This file is part of FAST-OAD : A framework for rapid Overall Aircraft Design
+#  Copyright (C) 2024 ONERA & ISAE-SUPAERO
+#  FAST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import openmdao.api as om
+import pytest
+
+from ..group import CycleGroup
+
+
+def test_base_cycle_group():
+    class Group1(CycleGroup):
+        pass
+
+    # -------------------------------------------------------------------------
+    problem = om.Problem()
+    problem.model.add_subsystem("group", Group1())
+    problem.setup()
+
+    assert isinstance(problem.model.group.linear_solver, om.DirectSolver)
+    assert isinstance(problem.model.group.nonlinear_solver, om.NonlinearBlockGS)
+
+    # -------------------------------------------------------------------------
+    problem = om.Problem()
+    problem.model.add_subsystem("group", Group1(use_solvers=False))
+    problem.setup()
+
+    assert isinstance(problem.model.group.linear_solver, om.LinearRunOnce)
+    assert isinstance(problem.model.group.nonlinear_solver, om.NonlinearRunOnce)
```

Comment:
I don't see you explicitely telling that the solver are the RunOnce variant, is it something OpenMDAO does on its own ?

Reply:
> I don't see you explicitely telling that the solver are the RunOnce variant, is it something OpenMDAO does on its own ?

Exactly. Default solvers in OpenMDAO groups are these.